presto-spi/src/main/java/com/facebook/presto/spi/block/BlockUtil.java

```diff
@@ -61,6 +59,13 @@ static void checkValidRegion(int positionCount, int positionOffset, int length)
         }
     }
 
+    static void checkValidPosition(int positionCount, int position)
```

Comment:
I think it's more natural to pass `position` first then `positionCount` as if asking: if `position` is correct in array with `positionCount` elements?

Reply:
I agree, but i wanted to be consistent with `BlockUtil#checkValidRegion`. Do you think we can change order arguments in that method too?

Reply:
Not that important

Reply:
@sopel39 i updated the signature

Reply:
Still LGTM :)